app/controllers/static_pages_controller.rb

```diff
@@ -1,5 +1,8 @@
 class StaticPagesController < ApplicationController
   def home
+    if !current_tenant
```

Comment:
[Should we use config to require tenants?](https://github.com/ErwinM/acts_as_tenant#configuration-options)

Reply:
we want a non-tenant top level to work, so people can browse all pets, but it won't be used much.

Reply:
But that line is just a hack - we'll need to think through how the UI behaves in that case.